src/pretix/base/services/orders.py

```diff
@@ -941,7 +941,10 @@ def _order_placed_email(event: Event, order: Order, pprov: BasePaymentProvider,
             log_entry,
             invoices=[invoice] if invoice and event.settings.invoice_email_attachment else [],
             attach_tickets=True,
-            attach_ical=event.settings.mail_attach_ical
+            attach_ical=event.settings.mail_attach_ical,
+            attach_other_files=filter(lambda a: a, [
+                event.settings.get('mail_attachment_new_order', as_type=str, default='')[7:]
```

Comment:
I know I often stumble upon those _magic_ numbers. Why do we need an offset here and why is it 7? If it is due to `file://` shouldn’t we strip the protocol off more explicitly?